### PR TITLE
Added feature to show completed todos.

### PR DESCRIPTION
**Pull request by swaindhruti**

### CODE REVIEW

**Review by sudo-rsingh — COMMENTED** [review]
- Looks good.
- Show me screenshot of it working.

**Comment by sudo-rsingh** [comment]
and resolve conflict in todo.go

**Comment by sudo-rsingh** [comment]
Issue Maintainer - @Meru45 
Assigned - @coder-XX04

**Comment by swaindhruti** [comment]
> * Looks good.
> * Show me screenshot of it working.
Screenshot 2023-10-16 035125

File: todo.go
Comment (Meru45) [inline todo.go]: No need of this footer here. The footer is supposed to be in main list where it shows the no of pending tasks. Here maybe we can show number of completed tasks in green.

File: todo.go
Comment (Meru45) [inline todo.go]: This line generates a warning that is open for modification.

File: todo.go
Comment (Meru45) [inline todo.go]: Why is there an unnecessary comment here?

File: cmd/todo/main.go
Comment (sudo-rsingh) [inline cmd/todo/main.go]: Do a `git pull` we have made some change to the location of file.

Comment (sudo-rsingh) [reply cmd/todo/main.go]: in another pr we have already did that.

File: cmd/todo/main.go
Comment (Meru45) [inline cmd/todo/main.go]: Maybe change this to showCompleted

**Review by Meru45 — CHANGES_REQUESTED** [review]

File: todo.go
Comment (Meru45) [reply todo.go]: on line 156

File: todo.go
Comment (Meru45) [inline todo.go]: Instead of 'CreatedAt,' it should be 'CompletedAt' here.

**Review by Meru45 — CHANGES_REQUESTED** [review]

**Comment by sudo-rsingh** [comment]
@coder-XX04 We can not merge until and unless the merge conflict is resolved.

**Pull request closed without merging**
